registry.json

```diff
@@ -4533,5 +4533,18 @@
     "telegram": "CrypTopo",
     "operatorName": "Alfred",
     "github": "avilsmeier"
+  },
+   {
+    "ticker": "ADACT",
+    "poolId": "22ab39540db22349b1934f5dcb7788440c33709ea9fdac07fb343395",
+    "social": {
+     "website": "https://adacapital.io",
+     "twitter": "ada_capital",
+     "telegram": "adacapital",
+     "youtube": "https://www.youtube.com/channel/UCnZQvrAsqteYLpmEWb418yA",
+     "medium": "ada-capital"
+    },
+    "operatorName": "Thomas",
```

Comment:
You must include the telegram key and it must be of an individual not a group or channel